lib/component/stringify.js

```diff
@@ -7,11 +7,12 @@ module.exports = R.curry(function stringify (opts, tree) {
     var preamble = [
         'var React = require(\'react\');',
         '',
-        'function ' + displayName + ' (props) {',
+        'var ' + displayName + ' = React.forwardRef(function(rawProps, ref) {',
```

Comment:
```suggestion
        'var forwardRef = React.forwardRef || function(c) { return c }'
        'var ' + displayName + ' = forwardRef(function(props, ref) {',
        '    props.ref = ref || props.ref;'
```

Reply:
@nowells Could you please implement the suggestion?

Reply:
The `var props = Object.assign({}, {ref: ref}, rawProps);` part is excess now